assets/app/views/builds.html

```diff
@@ -170,6 +187,9 @@
               Output image: {{build.parameters.output.dockerImageReference}}
             </div>            
           </div>
+          <div ng-show="true">
```

Comment:
hardcoded true here, assuming this was supposed to be checking if a build was already in progress for the build config?